lib/refocus/samples/collector.rb

```diff
@@ -23,6 +24,15 @@ def upsert_bulk
       end
       alias_method :submit, :upsert_bulk
 
+      def format_sample(name:, aspect:, value:nil, messageBody:nil, messageCode:nil, relatedLinks:nil)
+        sample = {name: "#{name}|#{aspect}"}
+        sample[:value] = value.to_s if value
+        sample[:messageBody] = messageBody.to_s if messageBody
+        sample[:messageCode] = messageCode.to_s if messageCode
+        sample[:relatedLinks] = relatedLinks if relatedLinks
+        sample
+      end
```

Comment:
Hey - I'm actually noticing some stuff I missed before too. 

Basically - we should be using snake_case for  the parameters, rather than camelCase, because Ruby :). https://github.com/bbatsov/ruby-style-guide#naming

Naturally, we have to pass camelcase along to the endpoint. But the params themselves should be named with snake case names.